Backpack/src/main/java/net/skyscanner/backpack/button/BpkButton.kt

```diff
@@ -91,43 +97,36 @@ open class BpkButton @JvmOverloads constructor(
 
   @Suppress("DEPRECATION")
```

Comment:
didn't see this before but I don't think we should suppress any deprecation, if we need it then let's keep the warning there so are aware of it.